pipeline/tasks/veneer_tasks.py

```diff
@@ -55,10 +55,10 @@ def _move_to(self, intermediate_config_location, veneer_api_yaml):
                         os.path.abspath(intermediate_config_location),
                         [os.path.abspath(c_out) for c_out in veneer_api_yaml])
 
-        if len(veneer_api_yaml) == 0:
+        if len(veneer_api_yaml) > 1:
+            raise ValueError(error_fmt + 'Multiple locations specified')
+        elif len(veneer_api_yaml) == 0:
             print error_fmt + 'No location specified'
```

Comment:
Updated. Note that this will cause consecutive runs of the config generator to fail, because the file will already exist. Will this be OK? I thought yes because we don't expect the config generator to be run repeatedly, just once and then make manual edits to the file.